src/cloudAdapter/genezio/genezioAdapter.ts

```diff
@@ -30,9 +32,49 @@ export class GenezioCloudAdapter implements CloudAdapter {
         }, cliProgress.Presets.shades_grey);
 
         const promisesDeploy = input.map(async (element) => {
-            if (element.unzippedBundleSize > BUNDLE_SIZE_LIMIT) {
-                throw new Error(`Class ${element.name} is too big: ${(element.unzippedBundleSize/1048576).toFixed(2)}MB. The maximum size is ${BUNDLE_SIZE_LIMIT/1048576}MB. Try to reduce the size of your class.`);
-            }
+            const { dependenciesInfo, allNonJsFilesPaths } = element;
+            
+                 if (element.unzippedBundleSize > BUNDLE_SIZE_LIMIT) {
+                    if (!dependenciesInfo) {
+                        throw new Error(
+                          `Your class ${element.name} is too big: ${element.unzippedBundleSize} bytes. The maximum size is 250MB. Try to reduce the size of your class.`
+                        );
+                    }
+
+                      const allfilesSize = await calculateBiggestFiles(
+                        dependenciesInfo,
+                        allNonJsFilesPaths
+                      );
+
+                   const table = new Table({
+                     head: ["Dependencies", "Non-JS Files"],
```

Comment:
change to: `head: ["BIggest Dependencies", "BIggest Non-JS Files"]`
and `Non-JS` is not correct because it is not language agnostic